physics/MP/Thompson/mp_thompson.F90

```diff
@@ -929,43 +891,6 @@ subroutine mp_thompson_run(ncol, nlev, con_g, con_rd,        &
          !  !vts1       => null()
          !  !txri       => null()
          !  !txrc       => null()
```

Comment:
What about these three?

Reply:
Can we get rid of them and then the entire block from line 929 to 970 (line numbers lhs)?

Reply:
@grantfirl Should I remove these commented lines Dom asks about.

Reply:
Yes, please. I'd do it, but I can't push to your branch.

Reply:
Removed.